### PR TITLE
pvs/V560: part of conditional expression is always false

### PR DESCRIPTION
**Pull request by ngortheone**
Fixes discovered by PVS issues [V560](https://www.viva64.com/en/w/v560/)

* makemap: same condition is used to skip outer loop cycle
* lints: spaces, braces

### CODE REVIEW

File: src/nvim/getchar.c [hunk 1]
```diff
@@ -4111,36 +4119,42 @@ makemap (
...
             return FAIL;
-          if (mp->m_nowait && fputs(" <nowait>", fd) < 0)
+          }
+          if (buf != NULL && fputs(" <buffer>", fd) < 0) {
             return FAIL;
-          if (mp->m_silent && fputs(" <silent>", fd) < 0)
+          }
+          if (mp->m_nowait && fputs(" <nowait>", fd) < 0) {
             return FAIL;
-          if (mp->m_noremap == REMAP_SCRIPT
```
Comment (ngortheone) [inline src/nvim/getchar.c]: same condition used to skip look cycle, line 3995

File: src/nvim/getchar.c [hunk 2]
```diff
@@ -3950,149 +3950,157 @@ void vim_unescape_csi(char_u *p)
...
+  char_u      c2;
+  char_u      c3;
   char_u      *p;
   char        *cmd;
-  int abbr;
-  int hash;
-  bool did_cpo = false;
+  int         abbr;
+  int         hash;
+  bool        did_cpo = false;
```
Comment (justinmk) [inline src/nvim/getchar.c]: just for future reference, we generally don't align declarations. So this is reverting a previous un-align.

**Comment by ngortheone** [comment]
I will undo this

On July 28, 2019 5:30:21 PM EDT, "Justin M. Keyes" <notifications@github.com> wrote:
>justinmk commented on this pull request.
>
>
>
>>    char_u      *p;
>   char        *cmd;
>-  int abbr;
>-  int hash;
>-  bool did_cpo = false;
>+  int         abbr;
>+  int         hash;
>+  bool        did_cpo = false;
>
>just for future reference, we generally don't align declarations. So
>this is reverting a previous un-align. 
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/neovim/neovim/pull/10641#pullrequestreview-267523356

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by justinmk** [comment]
This would be a case where a separate `lint` commit would help anyone needing to bisect in the future.

**Comment by ngortheone** [comment]
@justinmk split into 2 commits and removed declaration alignin

**Pull request merged**